bootstrap/src/index.ts

```diff
@@ -1,21 +1,16 @@
 import { Requester, logger } from '@chainlink/external-adapter'
-import { types } from 'util'
 import { withCache, defaultOptions, redactOptions } from './lib/cache'
 import * as util from './lib/util'
 import * as server from './lib/server'
 import * as gcp from './lib/gcp'
 import * as aws from './lib/aws'
-import {
-  ExecuteSync,
-  AdapterRequest,
-  ExecuteWrappedResponse,
-  AdapterHealthCheck,
-} from '@chainlink/types'
+import { ExecuteSync, AdapterRequest, Execute, AdapterHealthCheck } from '@chainlink/types'
 
-type Middleware = (execute: ExecuteWrappedResponse) => Promise<ExecuteWrappedResponse>
+type Middleware = (execute: Execute) => Execute
+type AsyncMiddleware = (execute: Execute) => Promise<Execute>
 
 // Try to initialize, pass through on error
-const skipOnError = (middleware: Middleware) => async (execute: ExecuteWrappedResponse) => {
+const skipOnError = (middleware: Middleware | AsyncMiddleware) => async (execute: Execute) => {
```

Comment:
I think we are good with having just one `Middleware` type, one that is async.

Reply:
Also a good point.